src/routes/_store/observers/bottomNavObservers.js

```diff
@@ -0,0 +1,8 @@
+export function bottomNavObservers (store) {
+  store.observe('bottomNav', bottomNav => {
+    // disables or enables the style
+    document.body.classList.toggle('bottom-nav', bottomNav);
+    document.querySelector('.main-content').classList.toggle('bottom-nav', bottomNav);
+    document.getElementById('main-nav').classList.toggle('bottom-nav', bottomNav);
```

Comment:
It's not clear to me why we need to set this in three different places.

Historically, the way I've done this is to toggle the entire `<style>` tag itself. You can see some examples here:

https://github.com/nolanlawson/pinafore/blob/b152359428c9582c0a212f0872cc805cfb8c08ad/src/build/template.html#L35-L44

https://github.com/nolanlawson/pinafore/blob/16e66346d7582ffddf27112706f834a4a0fbbbcc/src/routes/_store/observers/grayscaleObservers.js#L5-L16

Reply:
agreed. i'll have to pull up my sleeves and try to get my local env running so i can clean this up